src/routes/dashboard/(admin)/drafts/SystemLogsTab.svelte

```diff
@@ -44,13 +52,23 @@ Needs to distinguish the following events (one 'event' being a grouping of choic
   </label>
 </div>
 {#each events as [unix, choices], index (index)}
-  {@const labs = [...new Set(choices.map(({ labId }) => labId))]}
+  {@const labs = Array.from(
+    choices.reduce((set, { labId, round }) => set.add(`${labId}|${round}`), new Set<string>()),
+    key => {
+      const [labId, round] = key.split('|');
+      assert(typeof round !== 'undefined');
+      assert(typeof labId !== 'undefined');
+      return [labId, round === 'null' ? null : parseInt(round, 10)];
```

Comment:
This raises my eyebrows. I think we're better off serializing the `round` as an empty string rather than the string `'null'`.

Reply:
Yeah, I personally prefer `round.length === 0`, but they check the same idea.